packages/account/src/Sections/Security/ClosingAccount/closing-account-steps.tsx

```diff
@@ -49,9 +58,7 @@ const ClosingAccountSteps = observer(({ redirectToReasons }) => {
             {is_from_derivgo ? (
                 <div className='closing-account__buttons-container'>
                     <Button
```

Comment:
Since `Close my account` button is same in 2 places, can this be made as a component and reused?